apps/ocpp/admin/charger/utils.py

```
@@ -0,0 +1,8 @@
+"""Shared helper utilities for charger admin mixins."""
+
+from ..common_imports import *
+
+
+def charger_display_name(charger: Charger) -> str:
```

Ruff (0.15.0)

[error] `Charger` may be undefined, or defined from star imports

(F405)